src/ontology/mondo-edit.obo

```diff
@@ -584963,6 +584963,20 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8003" xsd:anyURI
 
+[Term]
+id: MONDO:1030009
+name: persistent atrial fibrillation
+def: "A disorder characterized by continuous atrial fibrillation that lasts for more than seven days." [https://orcid.org/0009-0009-6791-365X]
```

Comment:
Definition: 
This definition implies that "any disorder" that has continuous atrial fibrillation that lasts for more than seven days is a "persistent atrial fibrillation". This is not correct. 
The definition (genus-differentia) should use the most specific genus, which here should be "atrial fibrillation" (ie the parent term). 

dbxref: 
- Please add the PMID of the paper shared in the issue as a dbxref for this term. 
- this ORCID refers to a private record. Is this the correct orcid? If so, who's orcid is it?

Reply:
Sure, I'll update the definition. I've been asked to remove this kind of thing before so I did not include it here.

I added the PMID as an xref on the definition.
The user provided this orcid in a [previous request](https://github.com/monarch-initiative/mondo/issues/8003). I will remove it based on your PR summary comment.

Reply:
> I've been asked to remove this kind of thing before so I did not include it here.

We discussed this previously: what was asked to remove from the definition is the disease name itself (ie the term name itself) from the definition. For instance, the definition of "persistent atrial fibrillation" should not start with "Persistent atrial fibrillation is a ..."

The Genus, ie most often parent term name, is needed for the definition to follow the "genus-differentia" pattern.